app/MMB/src/scenes/timeline/events/DepartureTimelineEvent.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+import * as React from 'react';
+import idx from 'idx';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation } from '@kiwicom/mobile-localization';
+import { StyleSheet, Color, Text } from '@kiwicom/mobile-shared';
+
+import type { DepartureTimelineEvent as DeparturelineEventType } from './__generated__/DepartureTimelineEvent.graphql';
+import TimelineEvent from '../TimelineEvent';
+import TimelineEventIcon from '../TimelineEventIcon';
+
+type Props = {|
+  +data: DeparturelineEventType,
+|};
+
+function renderNote(duration: number): React.Element<typeof Text> {
```

Comment:
This looks like it could be a separate component in a seperate file

Reply:
I think this note is really specific to this timeline event. 
What could be abstracted in a different file would be this: 
```
const Note = (note) => <Text style={styles.note}>{note}</Text>
```
as it is used in several timeline events.